packages/playground/windows/Playground-win32 (Package)/Playground-win32-packaging.proj

```diff
@@ -64,5 +64,5 @@
   </PropertyGroup>
   <Import Project="..\..\..\..\vnext\PropertySheets\WinUI.props" />
   <Import Project="$(WapProjPath)\Microsoft.DesktopBridge.targets" />
-  <Import Project="..\packages\$(WinUIPackageName).$(WinUI3Version)\build\Microsoft.WinUI.AppX.targets" />
+  <Import Project="..\packages\$(WinUIPackageName).$(WinUI3Version)\build\Microsoft.WinUI.AppX.targets" Condition="Exists('..\packages\$(WinUIPackageName).$(WinUI3Version)\build\Microsoft.WinUI.AppX.targets')" />
```

Comment:
This is just a bug we have in playground I hit when creating and using this script, as the file doesn't exist before a nuget restore, which my script doesn't "technically" require.